respa_admin/templates/respa_admin/price_lists/_price_list_template.html

```diff
@@ -0,0 +1,37 @@
+{% load i18n %}
+{#TODO: Fix the classes that are copied from Units#}
+<section class="price-lists">
+    <div class="container">
+        <div class="unit-list container-fluid">
+            <ul class="list-group">
+                <li class="list-group-item sort-bar">
+                    <a class="col-4 btn order-by-link"
+                       href="?order_by={% if order_by == "name" %}-name{% else %}name{% endif %}">
+                        {% trans "Name" %} <span class="glyphicon glyphicon-sort" aria-hidden="true"></span>
+                    </a>
+                    <div class="order-by-link"></div>
+                    <div class="order-by-link"></div>
+                    <div class="order-by-link"></div>
+                </li>
+                {% for template in price_list_templates %}
+                    <li class="list-group-item">
+                        <div class="panel panel-default">
+                            <div class="panel-body">
+                                <div class="col-4">
+                                    <h4>{{ template.name }}</h4>
+                                </div>
+                                <div class="col-4">
+                                    <h6>
+                                        <span class='glyphicon glyphicon-plus plus' aria-hidden="true"></span>
+                                        <a href="{% url 'respa_admin:new-price-list-from-template' template.pk %}">{% trans 'Create price list' %}</a>
```

Comment:
"Create price list" doesn't seem to have a Finnish translation. There's a translation for "Create a new price list", though.

Reply:
Yeah, it should just be "Create a new price list".